modules/raft-client/src/main/java/org/apache/ignite/raft/client/message/ActionRequest.java

```diff
@@ -35,19 +35,30 @@
      */
     Command command();
 
+    /**
+     * @return {@code True} for linearizable reading.
+     */
+    boolean readOnlySafe();
```

Comment:
Currently there are no usages of readOnlySafe(). Why we need it? In Which cases do you expect to use non-linearizable readings? Besides that it's a bit confusing cause readOnlySafe only has sense for read operations, but ActrionRequest both covers reads and writes. In other words within given abstaction it's possible to create write request with readOnlySafe set to false. Do we expect non-linearizable writes?

Reply:
Usages are in jraft branch.

Reply:
This concept comes from RAFT linerializable reading guarantees, which require stable leader before reading. 
It makes no sense for single node RAFT cluster, so no getter usage here.
You can find actual usage in ignite-13885